### DISCUSSION TITLE
Could not load file or assembly System.Web.Services in oqtane module

### DISCUSSION THREAD

**Discussion by pnicosia80** [oqtane/oqtane.framework#4282] [Q&A]
Good evening,
in my oqtane module, within the server project, I need to reference a library that uses the service model, therefore which contains references to System.Web.Services.

Unfortunately when running I get this error:

Could not load file or assembly 'System.Web.Services, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. Reference assemblies cannot be loaded for execution. (0x80131058).

Obviously the module project is in .Net Core and I'm trying to use a .Net Framework dll, I understand it, but I can't convert the library from .Net Framework to .Net Standard.
What other workaround do you recommend??
The management software we use uses SOAP services and so it becomes impossible to continue in Oqtane.

Thanks

**Comment by thabaum** [comment]
@pnicosia80 I did some browsing and found this for connecting Blazor to SOAP:
[https://stackoverflow.com/questions/59231073/how-do-i-connect-blazor-and-soap-api](https://stackoverflow.com/questions/59231073/how-do-i-connect-blazor-and-soap-api)

**Comment by sbwalker** [comment]
@pnicosia80 this type of error is usually caused when the DLL you are referencing in your module has not been copied to the Oqtane.Server /bin folder. The debug.cmd file in your module's Package project should include references to all assemblies that your module relies on so that they are automatically copied to the Oqtane /bin when you compile your module.

**Reply by pnicosia80** [reply]
I did this, but the error persists. This dll was also present in the oqtane bin before. Could it be the incompatibility with .NET Core??
